packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Cover.Example.tsx

```diff
@@ -46,20 +111,20 @@ export class CalloutCoverExample extends React.Component<{}, ICalloutCoverExampl
     // Large beak will disable some position to avoid beak over the callout edge.
     return (
       <div className="ms-CalloutExample">
```

Comment:
Can you also remove the remaining classes like this which weren't doing anything? (in all the files)

Reply:
Removed! I replaced some with fragments but I don't know if it's a recommended practice in this codebase?